test/gcloud_speech_to_text/integration_test.exs

```diff
@@ -32,9 +35,10 @@ defmodule Membrane.Element.GCloud.SpeechToText.IntegrationTest do
             interim_results: false
           ] ++ recognition_opts
         )
+      )
     ]
 
-    Testing.Pipeline.start_link(links: Membrane.ParentSpec.link_linear(children))
+    Testing.Pipeline.start_link(construct: links)
```

Comment:
In migration guide it was advised to change this function to `Testing.Pipeline.start_link_supervised!`, not that it makes any difference in functionality.